wfe/web-front-end.go

```diff
@@ -42,6 +42,8 @@ const (
 
 	// StatusRateLimited is not in net/http
 	StatusRateLimited = 429
+	// statusBadNonce is used to force sendError to send the proper error type
+	statusBadNonce = 0400
```

Comment:
Can you explain more about this? It looks like you are using an octal constant `0400`, which evaluates to 256 in decimal, as a special sentinel value? If we require a sentinal value here I'd prefer it be in decimal and more clearly out of normal range, e.g. `1400`.

Jeff's working on a refactor to how we pass through ProblemDetails that should make this class of issues easier to deal with.